ocdaction/ocdaction/templates/core/contact.html

```diff
@@ -5,6 +5,23 @@
 {% block main %} 
     <section>
  		<h1>Contact</h1>
-    </section> 
+        <div class="col-xs-offset-2 col-xs-8 col-sm-offset-3 col-sm-6 col-md-offset-4 col-md-4 col-lg-offset-4 col-lg-4">
+        <form role="form" method="post">
+         {% csrf_token %}
+         {{ form.as_p }}
+        <div class="row">
+        <div class="col-xs-12">
+        <label for="email">Email: </label>
+        <input id="email" type="text" name="email" value="{{ email }}">        
```

Comment:
this input should be of type email so that the user has to enter an email address, please remove the {{ email }}  this is just a static and this has no value form django anyway